packages/rollup-tests/src/status.json

```diff
@@ -1,7 +1,7 @@
 {
   "failed": 0,
   "skipFailed": 0,
-  "ignored": 377,
+  "ignored": 384,
   "skipped": 0,
-  "passed": 552
+  "passed": 545
```

Comment:
Seems some tests regressed, @underfin maybe  you have some idea to fix this?

Reply:
It's tracked in https://github.com/rolldown/rolldown/issues/2569. I investigate a little bit and don't think this PR is the root cause.

Reply:
Yeah, I don't mean say this pr causes it, since @underfin take charge in this part, so I left him to take a look.

Reply:
Get it. Just add some notes.